specification/signalr/resource-manager/Microsoft.SignalR/preview/2018-03-01-preview/signalr.json

```diff
@@ -0,0 +1,1034 @@
+                    "readOnly": true
+                },
+                "tags": {
+                    "description": "Tags of the service.",
+                    "type": "object",
+                    "additionalProperties": {
+                        "type": "string"
+                    }
+                },
+                "sku": {
+                    "$ref": "#/definitions/ResourceSku",
+                    "description": "SKU of the service."
+                },
+                "properties": {
+                    "$ref": "#/definitions/SignalRResponseProperties",
+                    "description": "The properties of the service.",
+                    "x-ms-client-flatten": true
+                }
+            }
+        },
+        "ResourceSku": {
+            "description": "The billing information of the resource.(e.g. basic vs. standard)",
+            "required": [
+                "name"
+            ],
+            "type": "object",
+            "properties": {
+                "name": {
+                    "description": "The name of the SKU. This is typically a letter + number code, such as A0 or P3.  Required (if sku is specified)",
+                    "type": "string"
+                },
+                "tier": {
+                    "description": "The tier of this particular SKU. Optional.",
+                    "enum": [
+                        "Free",
+                        "Basic",
+                        "Premium"
+                    ],
+                    "type": "string",
+                    "x-ms-enum": {
+                        "name": "SignalRSkuTier",
+                        "modelAsString": true
+                    }
+                },
+                "size": {
+                    "description": "Optional, string. When the name field is the combination of tier and some other value, this would be the standalone code.",
+                    "type": "string"
+                },
+                "family": {
+                    "description": "Optional, string. If the service has different generations of hardware, for the same SKU, then that can be captured here.",
+                    "type": "string"
+                },
+                "capacity": {
+                    "format": "int32",
+                    "description": "Optional, integer. If the SKU supports scale out/in then the capacity integer should be included. If scale out/in is not \r\npossible for the resource this may be omitted.",
+                    "type": "integer"
+                }
+            }
+        },
+        "SignalRResponseProperties": {
+            "description": "A class that describes the properties of the SignalR service.",
+            "type": "object",
+            "properties": {
+                "provisioningState": {
+                    "description": "Provisioning state of the resource.",
+                    "type": "string"
+                },
+                "status": {
```

Comment:
In the examples, this value is `"0"`. This should probably be an enumeration of values.

Reply:
(good example of use of enum:  https://github.com/Azure/azure-rest-api-specs/blob/master/specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/stable/2017-10-01/containerregistry.json#L1139 )